iOS Client/Store/MBStore.swift

```diff
@@ -96,42 +150,69 @@ class MBStore: NSObject {
         performDownload(clientFunction: client.getArticlesWithCompletion, persistentContainer: persistentContainer, deserializeFunc: MBArticle.deserialize, completion: completion)
     }
     
-    private func loadDevotions(completion: @escaping ([MBDevotion]?, Error?) -> Void) {
-        
-        let manager = FileManager.default
-        guard let url = manager.urls(for: .documentDirectory, in: .userDomainMask).first as URL? else {
-            completion(nil, MBDeserializationError.contextInsertionError(msg: "Could not create Devotions path"))
-            return
-        }
-        let path = url.path.appending("/devotions")
-        let pathUrl = URL(fileURLWithPath: path)
-        if !FileManager.default.fileExists(atPath: path) {
-            client.getDevotionsWithCompletion { data, error in
-                if error == nil, let data = data {
-                    self.save(data, pathUrl) { error in
-                        if error != nil {
-                            print("Could not save devotions data to disk")
-                        }
+    func markDevotionAsRead(date: String, completion: @escaping (Error?) -> Void) {
```

Comment:
why does this use a completion block? Can't it just return an error type directly since there's nothing async about it?